setup.cfg

```diff
@@ -47,6 +47,7 @@ packages =
 console_scripts = 
     deepaas-run = deepaas.cmd.run:main
     deepaas-wsk = deepaas.cmd.wsk:main
+    deepaas-execute = deepaas.cmd.execute:main
```

Comment:
Now I realize: Maybe it is better to name it "deepaas-predict" rather than "deepaas-execute".

Reply:
I agree. I make the change.